src/templates/devinfo.handlebars

```diff
@@ -160,10 +160,10 @@
         {{/ne_and}}
     {{/if}}
     </h3>
-    <ul>
+    <ul id="contributors--readonly" class="userpage-link">
```

Comment:
@bsstoner No specific reason other than I did a bit of back and forth with that end ended up leaving the class in the parent element. Updated, thanks!